engines/python/setup/djl_python/deepspeed.py

```diff
@@ -236,17 +301,71 @@ def infer_task_from_model_architecture(self, config: PretrainedConfig):
                 f"Task could not be inferred from model config. "
                 f"Please manually set `task` in serving.properties.")
 
+    def get_model_pretrained(self, model_id_or_path, torch_dtype='auto', **kwargs):
+        tokenizer = AutoTokenizer.from_pretrained(model_id_or_path)
+        model = TASK_TO_MODEL[self.task].from_pretrained(model_id_or_path, torch_dtype=torch_dtype, **kwargs)
+        return model, tokenizer
+
+    def get_model_from_config(self, model_id_or_path, **kwargs):
+        tokenizer = AutoTokenizer.from_pretrained(model_id_or_path)
```

Comment:
please add some logic to allow `trust_remote_code` option

Reply:
`revision`

Reply:
Same as above